app/include/drivers/behavior.h

```diff
@@ -19,8 +20,7 @@
  * (Internal use only.)
  */
 
-typedef int (*behavior_keymap_binding_callback_t)(struct device *dev, u32_t position, u32_t param1,
-                                                  u32_t param2);
+typedef int (*behavior_keymap_binding_callback_t)(struct zmk_behavior_binding *binding, struct zmk_behavior_binding_data data);
 typedef int (*behavior_sensor_keymap_binding_callback_t)(struct device *dev, struct device *sensor,
                                                          u32_t param1, u32_t param2);
```

Comment:
If we're going to do this, let's refactor this callback to `(struct zmk_behavior_binding *binding, struct device *sensor)` while we're at it.